crypto/x509/v3_san.c

```diff
@@ -434,10 +434,9 @@ static int copy_email(X509V3_CTX *ctx, GENERAL_NAMES *gens, int move_p)
         return 0;
     }
     /* Find the subject name */
-    if (ctx->subject_cert)
-        nm = X509_get_subject_name(ctx->subject_cert);
-    else
-        nm = X509_REQ_get_subject_name(ctx->subject_req);
+    nm = ctx->subject_cert != NULL ?
```

Comment:
```suggestion
    nm = (ctx->subject_cert != NULL) ?
```
Helps readability and for those that don't rembmer the precedence order of comparison vs. ternary vs. assignment

Reply:
We do not do this kind of extra parentheses elsewhere. Please don't start it arbitrarily here.